wgpu-core/src/command/compute.rs

```diff
@@ -490,6 +497,19 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         .map_pass_err(scope)?;
                     check_buffer_usage(indirect_buffer.usage, BufferUsage::INDIRECT)
                         .map_pass_err(scope)?;
+
+                    let begin_offset = offset;
+                    let end_offset = offset + mem::size_of::<wgt::DispatchIndirectArgs>() as u64;
+                    if end_offset > indirect_buffer.size {
+                        return Err(ComputePassErrorInner::IndirectBufferOverrun {
+                            offset,
+                            begin_offset,
```

Comment:
why are we having separate `begin_offset` from `offset`? They appear to bbe the same thing

Reply:
that was caused by ~mindless copying~ style preserving from render.rs where this is done in two different places 😄
I'll clean all of them up then 🤷